hazelcast/src/main/java/com/hazelcast/internal/ascii/rest/HttpCommandProcessor.java

```diff
@@ -79,6 +79,7 @@
     // WAN
     public static final String URI_WAN_SYNC_MAP = URI_WAN_BASE_URL + "/sync/map";
     public static final String URI_WAN_SYNC_ALL_MAPS = URI_WAN_BASE_URL + "/sync/allmaps";
+    public static final String URI_WAN_SYNC_GET_PROGRESS = URI_WAN_BASE_URL + "/sync/getProgress";
```

Comment:
The thing with our get endpoints are I don't know how can I send the data to the cluster. We need to send the sync uuid in rest request. With POST, I can put this in the payload. However with GET there is no example (at least that I can find) of sending information to the cluster.

Normally we could have something like this: `sync/progress/syncUUID` but I'm not so sure since we have nothing like this in our REST API previously. Given that all of our WAN requests are POST I think it will be inconsistent. But I'm also not sure which is the best. WDYT?

Reply:
It seems I missed this https://docs.hazelcast.com/hazelcast/5.2/maintain-cluster/rest-api#retrieving-entries-from-a-map. We do indeed pass map name in url here.

Reply:
Done.